src/main/java/elements/Teleportable.java

```diff
@@ -0,0 +1,5 @@
+package elements;
+import game.Player;
+
+public interface Teleportable {
```

Comment:
See the Game.java suggestion. If you are modifying playerState (which now includes the player position), then you can simply put the logic for changing the player position in the modifyPlayerState function that comes for free with the Interactable interface.

Reply:
resolved in commit 41df858